src/controllers/revision.ts

```diff
@@ -126,7 +126,7 @@ export const getRevisionPreview = async (req: Request, res: Response, next: Next
       logger.error('Something went wrong trying to download file from data lake');
       throw err;
     }
-  } else if (revision.onlineCubeFilename.includes('protocube')) {
+  } else if (revision.onlineCubeFilename?.includes('protocube')) {
```

Comment:
@j-maynard is onlineCubeFilename only created once the revision is published?

Reply:
this can be seen on dev when looking at https://dev.stats.cymru/en-GB/publish/0c8164e6-dfb4-493c-8fe5-7154206160f5/cube-preview

Reply:
Nope...  We create a protocube as soon as we've validated the initial data table.   A protocube only has the fact table in... not any of the lookup tables or views.  It saves us having to try to load in all the data every time we want a preview.

Reply:
So actually now the only time this should be `null` is if we've failed to load the fact table after the preview.